class-components/src/scripts/types/interfaces.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode } from 'react';
```

Comment:
Please split the contents of this file and move all the types to their respective component folders